systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaResource.java

```diff
@@ -46,7 +46,7 @@
     }
 
     public static DoneableKafka kafkaEphemeral(String name, int kafkaReplicas) {
-        return kafkaEphemeral(name, kafkaReplicas, 3);
+        return kafkaEphemeral(name, kafkaReplicas, Math.min(kafkaReplicas, 3));
```

Comment:
Why are we using the `min(...)` here and on line 58? I think it deserves some explanation.

Reply:
AFAIK, It is because sometimes, we deploy Kafka cluster with only 1 Kafka broker and if We did not specify Zookeeper replicas it will create the cluster with `1 Kafka and 3 zookeepers`.